### PR TITLE
Add multiplier to company page

### PR DESCRIPTION
**Pull request by EyalMecilati**

### CODE REVIEW

File: src/comps/companyPage/companyPage.tsx [hunk 1]
```diff
@@ -19,11 +28,54 @@ function CompanyPage() {
...
   };
-  const [company, setCompany] = useState({} as Company);
+
+  const [company, setCompany] = useState({} as CompanyTemp);
+  const [companySummery, setCompanySummery] = useState({} as CompanySummery)
+
   useEffect(() => {
-    getCompany(params.id).then((data) => setCompany(data));
+    getCompany(params.id).then((data) => {
+  const companyFF = convert(data);
```
Comment (alissaVrk) [inline src/comps/companyPage/companyPage.tsx:37]: it's nicer to do in the CompanyDB

File: src/comps/companyPage/projectsPage.tsx [hunk 2]
```diff
@@ -134,6 +143,9 @@ export default function ProjectsPage({
               product={getCurrencyAndProduct(project.currency,product).productType}
             />
           ) : null}
+          {!isEmpty(project.deposits?.items) ? (
+          <DepositsSummery summery={summery} printDescriptionsItemByValue={printDescriptionsItemByValue}/>
```
Comment (alissaVrk) [inline src/comps/companyPage/projectsPage.tsx:147]: it isn't a summery per project, it's for the whole company
don't know what Tal wanted but this is what you have in the summery

**Review by alissaVrk — CHANGES_REQUESTED** [review]

File: src/comps/companyPage/projectsPage.tsx (hunk 2 above)
Comment (EyalMecilati) [reply src/comps/companyPage/projectsPage.tsx:147]: he wanted to add a panel for Deposits Summery before the deposits panel

File: src/logic/companies/CompaniesDB.ts [hunk 3]
```diff
@@ -36,6 +37,10 @@ import * as companyMutations from "./db/companyMutations";
 import * as companyQueries from "./db/companyQueries";
 import convertCompany from "./companyConverter";
 import moment from "moment";
+import { convertToSummery } from "../summery/convertToSummery";
```
Comment (alissaVrk) [inline src/logic/companies/CompaniesDB.ts:40]: you can't convert to summery here...

File: src/logic/companies/CompaniesDB.ts [hunk 4]
```diff
@@ -36,6 +37,10 @@ import * as companyMutations from "./db/companyMutations";
...
+import convert from "./companyConverter";
+import { currencies, productPrices } from "../currencyAndProductPrices";
+import { selectedProduct } from "../summery";
```
Comment (alissaVrk) [inline src/logic/companies/CompaniesDB.ts:43]: so is this..

**Review by alissaVrk — CHANGES_REQUESTED** [review]

**Review by alissaVrk — APPROVED** [review]

**Pull request merged**